docker-compose.yml

```diff
@@ -28,3 +28,11 @@ services:
     container_name: dotnet
     ports:
       - "3000"
+  java:
+    build:
+      context: ./proxies/java
+      args:
+        - JAVA_SDK_VERSION=${JAVA_SDK_VERSION}
+    container_name: java
+    ports:
+      - "8080"
```

Comment:
why `8080`?

Reply:
couldnt figure out how to get 3000 to work

Reply:
its working on 3000 now